#pragma once

#include "public.h"

#include <yt/yt/core/bus/public.h>

namespace NYT::NRpc::NBus {

////////////////////////////////////////////////////////////////////////////////

IServerPtr CreateBusServer(NYT::NBus::IBusServerPtr busServer);

////////////////////////////////////////////////////////////////////////////////

} // namespace NYT::NRpc::NBus
